test/wdio_functional/helpers/helpers.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export function stripRequest(requestString) {
+  return requestString
+    .trim()
+    .replace(/\n|\r/g, '')
+    .replace(/\s+/g, '');
+}
+
+export function createTestSubjectSelectorCss(selector) {
```

Comment:
This is what the `@kbn/test-subj-selector` package is for.

Reply:
fwiw I find that package indirection unneccessary.  What is the benefit of a having a few helper functions bundled up in a package?  Rather than in some helper function file?  

Although I agree for the time being they should probably both use the package, or both use a new helper file.

Reply:
The syntax supported by the package is more than is implemented here, and I while agree it doesn't really justify the existence of a package that's what we are using everywhere else for all  tests.